build/windows/installer/livenessprobe/livenessprobe.cpp

```diff
@@ -113,13 +113,20 @@ int _tmain(int argc, wchar_t *argv[])
         wprintf_s(L"ERROR:Process:%s is not running\n", argv[1]);
         return NO_FLUENT_BIT_PROCESS;
     }
+    const DWORD bufferSize = 256;
+    wchar_t enableCustomMetricsValue[bufferSize];
```

Comment:
Do we need 256 bytes for value of environment variables which can true or false or "", right? This is static allocation and essentially, we are reserving 512 bytes here for both environment variables.